config.json

```diff
@@ -40,7 +40,7 @@
       "description": "MsBuild logging options.",
       "valueType": "passThrough",
       "values": [],
-      "defaultValue": "/l:BinClashLogger,Tools/net46/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log"
```

Comment:
@ericstj Could you take a look at the failures on this PR after the BinClashLogger update to config.json: https://ci.dot.net/job/dotnet_coreclr/job/master/job/x64_release_windows_nt_pri1_prtest/6380/consoleFull.  Does that indicate that CoreCLR's attempt to use BinClashLogger before wasn't actually doing anything?  I notice that CoreFx's config.json doesn't use the MsBuildEventLogging property.  Is our config.json stale?